### PR TITLE
Support driveItem for fields feature

### PR DESCRIPTION
**Pull request by Chentong-Xiang**

### CODE REVIEW

**Review by njerigrevious — APPROVED** [review]
Just finished reviewing, this looks great - good to go!

File: concepts/search-concept-files.md [hunk 1]
```diff
@@ -433,6 +431,67 @@ Content-type: application/json
 }
 ```
 
+### driveItem request
```
Comment (Lauragra) [inline concepts/search-concept-files.md:434]: Please check this response on the review site and fix the formatting; it isn't rendering correctly. All else looks good!

Comment (Chentong-Xiang) [reply concepts/search-concept-files.md:434]: Fixed the format! Thanks.

**Review by njerigrevious — APPROVED** [review]
Thanks for making the final fixes, this is good to go!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**